package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "poa-parity-bridge-contracts",
+  "name": "token-bridge-contracts",
   "version": "1.0.0",
```

Comment:
does it make sense to update the version too? Is it possible to make it auto upgradable if we tag a new release?